### PR TITLE
feat(@angular/cli): support WSL (Windows Subsystem for Linux)

### PR DESCRIPTION
**Pull request by ciesielskico**
Updates opn to 5.0.0, which supports opening apps in Windows from WSL.
Useful for the `-o` option.

Fixes https://github.com/angular/angular-cli/issues/6012

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "minimatch": "^3.0.3",
     "node-modules-path": "^1.0.0",
     "nopt": "^4.0.1",
-    "opn": "4.0.2",
+    "opn": "5.0.0",
```
Comment (clydin) [inline package.json]: can probably go with `~5.0.0` here.

Comment (ciesielskico) [inline package.json]: Done

**Comment by clydin** [comment]
Also need to adjust https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/package.json#L61

And commit message prefix should be "feat(@angular/cli)".

**Comment by ciesielskico** [comment]
I don't understand? That line is adjusted through the commit. 
Or do you mean that I should revert the last commit and rephrase it?

**Comment by clydin** [comment]
There are two different package.json's that need to be changed.  The top level one (which you changed) is used for CLI development and the one I linked to above is the one published and used by the end user.

**Comment by ciesielskico** [comment]
Right, got it.

**Comment by filipesilva** [comment]
Heya, thanks for making this PR! Can you rebase it on master please? We had some limitations on what could go on master a few days ago so CI failed, but now it should be fine.

**Comment by ciesielskico** [comment]
@filipesilva I just rebased on master on another machine and somehow it does not recognize the commits as myself. Could you tell me what to do to get the googlebot to accept the PR?

**Comment by filipesilva** [comment]
@googlebot everything is fine, it was just a messy rebase.

**Comment by hansl** [comment]
I verified that @ciesielskico has signed the CLA.

**Comment by hansl** [comment]
The author of these commits is `alexander@ciesielski.co`, which has NOT signed the CLA. Also, my commit should not be there.

@ciesielskico you used the email `alexxde@gmail.com` to sign the CLA and it might be the one linked to your github account, which is why your name doesn't show up.

**Comment by ciesielskico** [comment]
Actually I have both emails linked with my Github, but I've had them set to private, which may be the reason the CLA didn't recognize. I've updated my profile, can you check again?

**Comment by hansl** [comment]
If you can re push your branch that would trigger a check. Change a file, add a comment or something, then push. Thanks.

**Comment by clydin** [comment]
FYI. `git commit --amend --no-edit` usually does the trick.

**Pull request closed without merging**